lib/Imagine/Vips/Imagine.php

```diff
@@ -203,6 +203,7 @@ protected function getLoadOptions($loader, $loadOptions = [])
                 break;
             case 'VipsForeignLoadGifFile':
             case 'VipsForeignLoadGifBuffer':
+            case 'VipsForeignLoadNsgifBuffer':    
```

Comment:
What about `VipsForeignLoadNsgifFile`? Would that exist, too? Can't check right now by myself

Reply:
Yes, could you please also add `VipsForeignLoadNsgifFile` here?

![image](https://github.com/rokka-io/imagine-vips/assets/34705285/7303460a-8396-4f8c-be04-27c6df6b809c)

Thanks!